### ISSUE TITLE
Switch Lugar to using production site

### ISSUE THREAD

**Issue by beamalsky** [datamade/committee-oversight#141]
- [x] Set up crontasks to run there (Ie, reverse the change made in #140)
- [x] Deploy a tagged commit
- [x] Sync production site db to match what's on the staging site

To restore the production database from the latest staging backup, I followed these steps.

1. Created a new backup in [this folder](https://s3.console.aws.amazon.com/s3/buckets/datamade-postgresql-backups/hearings/?region=us-east-1&tab=overview) by running:

```
ssh ubuntu@committeeoversight.datamade.us
(pg_dump -Fc -U postgres -d hearings | /usr/bin/aws s3 cp - s3://datamade-postgresql-backups/hearings/$(date -d "today" +"\%Y\%m\%d\%H\%M").dump) && echo "staging backup $(date -d "today" +"\%Y\%m\%d\%H\%M").dump complete" >> /tmp/committee-oversight-crontasks.log 2>&1
```

2. Visited [the folder](https://s3.console.aws.amazon.com/s3/buckets/datamade-postgresql-backups/hearings/?region=us-east-1&tab=overview) and downloaded the newest file into my `committee-oversight` root project directory as `hearings.dump`

3. Move `hearings.dump` to the production server, SSH into the server, and restore the database:

```
scp ~/Desktop/committee-oversight/hearings.dump ubuntu@committeeoversight-production.datamade.us:/tmp
ssh ubuntu@committeeoversight-production.datamade.us
sudo su postgres
dropdb hearings
createdb hearings
pg_restore -C -j4 --no-owner -U postgres -d hearings /tmp/hearings.dump | psql
```

- [x] Delete crontask from staging site

```
ssh ubuntu@committeeoversight.datamade.us
sudo rm /etc/cron.d/committee-oversight-crontasks
```

**Comment by beamalsky** [comment]
To restore the production database from the latest staging backup, I followed these steps.

1. Created a new backup in [this folder](https://s3.console.aws.amazon.com/s3/buckets/datamade-postgresql-backups/hearings/?region=us-east-1&tab=overview) by running:

```
ssh ubuntu@committeeoversight.datamade.us
(pg_dump -Fc -U postgres -d hearings | /usr/bin/aws s3 cp - s3://datamade-postgresql-backups/hearings/$(date -d "today" +"\%Y\%m\%d\%H\%M").dump) && echo "staging backup $(date -d "today" +"\%Y\%m\%d\%H\%M").dump complete" >> /tmp/committee-oversight-crontasks.log 2>&1
```

2. Visited [the folder](https://s3.console.aws.amazon.com/s3/buckets/datamade-postgresql-backups/hearings/?region=us-east-1&tab=overview) and downloaded the newest file into my `committee-oversight` root project directory as `hearings.dump`

[To be continued]

**Comment by beamalsky** [comment]
Interlude: I'm expanding the EC2 root volume following these steps

https://aws.amazon.com/premiumsupport/knowledge-center/expand-ebs-root-volume-windows/
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/recognize-expanded-volume-linux.html

**Comment by beamalsky** [comment]
The database backups haven't been complete with CMS content, I suspect because the tables in `hearings` are all owned by postgres on the production site. Meanwhile, on staging, all tables but ___ are owned by datamade (though views are owned by postgres).

I made a backup from the staging site run by the postgres user, and when I loaded it in locally it had the Wagtail content I'd expect to see from staging. 

So, I sshed into the production site and ran the following:

```bash
ssh ubuntu@committeeoversight-production.datamade.us
sudo su postgres
psql hearings
```

```psql
SELECT 'ALTER TABLE '|| tablename ||' OWNER TO datamade;' FROM
pg_tables WHERE schemaname = 'public';
```

...and ran the list of ALTER queries printed by the above command.
